operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -663,11 +663,11 @@ func (dc *CassandraDatacenter) GetContainerPorts() ([]corev1.ContainerPort, erro
 			ContainerPort: 8609,
```

Comment:
I think we should put a `tls` toggle on the cassdc, maybe with a secret name input too

Reply:
Ok, that's not a change I made, but happy to do so. Can this be a future PR?